docs/examples/xkcd_scientific_papers.md

```diff
@@ -0,0 +1,34 @@
+# xkcd scientific paper classifier
+
+[![](https://imgs.xkcd.com/comics/types_of_scientific_paper_2x.png){width="550"}](https://xkcd.com/2456/)
+
+
+!!! example "What kind of paper is 'Attention is All You Need'?"
+
+    This example extracts the types of papers from the comic and uses them to classify the paper "[Attention is All You Need](https://arxiv.org/abs/1706.03762)".
+
+    ```python
+    import marvin
+
+
+    # extract labels from the comic
+    paper_types = marvin.beta.extract(
+        data=marvin.beta.Image(
+            "https://imgs.xkcd.com/comics/types_of_scientific_paper_2x.png"
+        ), 
+        instructions="Extract the types of papers from the comic",
+    )
+
+    
+    # classify the paper
+    paper_text = load_paper("https://arxiv.org/pdf/1706.03762.pdf")
```

Comment:
should `load_paper` be defined here or is this meant to be a placeholder?

Reply:
placeholder, don't want to crowd the example. I will add a note.